pyproject.toml

```diff
@@ -170,14 +170,46 @@ requires = [
     "nn: mark a test as module test"
 ]
 
+[tool.coverage.run]
+branch = true
+source = [
+    'kornia/',
+]
+omit = [
+    '*/__main__.py',
+    '*/setup.py',
+]
+
 [tool.coverage.report]
-exclude_lines = [
-  "def __repr__",
-  "if __name__ == .__main__.:",
-  "if 0:",
-  "if self.debug:",
-  "pragma: no cover",
-  "raise",
+show_missing = true
+skip_covered = true
+fail_under = 87
```

Comment:
This is the actual coverage %, so if someone adds some code and doesn't add tests, the tool reports a percentage lower than this, then the coverage job will fail, and the whole matrix will fail too (collector job)

Reply:
We should add as milestone to be above 90